terragrunt/aws/cloudfront/s3.tf

```diff
@@ -0,0 +1,5 @@
+module "log_bucket" {
+  source            = "github.com/cds-snc/terraform-modules?ref=v0.0.47//S3_log_bucket"
```

Comment:
Can you bump the version ref on this to the latest `cds-snc/terraform-modules` release.